test/dotnet-new3.UnitTests/DotnetNewSearch.cs

```diff
@@ -871,7 +871,7 @@ private List<List<string>> ParseTableOutput(string stdOut, string[] expectedColu
             int headerLineIndex = Array.FindIndex(lines, line => expectedColumns.All(column => line.Contains(column)));
             string headerLine = lines[headerLineIndex];
             //table ends after empty line
-            int lastLineIndex = Array.FindIndex(lines, headerLineIndex + 1, line => line.Length == 0) - 1;
+            int lastLineIndex = Array.FindIndex(lines, headerLineIndex + 1, line => (line.Length == 0 || line.Contains("[Debug]"))) - 1;
```

Comment:
Will this start becoming a problem if the line after the table changes for some reason? Do we know that it will always be `[Debug]`?

Reply:
In case someone tracks this line later using "git blame", I think it would be nice to have a small explanation in PR description why we are making this change. I learned it from another PR, but not sure which one will end up in `git blame`.